client/src/TextInput.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+
+var inputStyle =
+  "bg-gray-600 placeholder-gray-400 px-5 py-3 rounded text-lg border border-gray-400";
+
+class TextInput extends Component {
```

Comment:
We might want to consider sticking with a functional component here, unless we have a good reason to switch to a class. This is an interesting SO question that might be helpful - https://stackoverflow.com/questions/36097965/when-to-use-es6-class-based-react-components-vs-functional-es6-react-components

Reply:
Understandable, and it makes sense after reading on StackOverflow about the component.  As they say, technology is changing. Thank you for sharing the link.